bootstrap/kubeadm/main.go

```diff
@@ -75,6 +78,15 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 
+	fs.DurationVar(&leaseDuration, "lease-duration", 15*time.Second,
+		"Interval at which non-leader candidates will wait to force acquire leadership (e.g. 20s)")
```

Comment:
The output looks a little funny:

```
chuckh-a02:kubeadm cha$ ./kubeadm --help
Usage of ./kubeadm:
*snip*
      --lease-duration duration               Interval at which non-leader candidates will wait to force acquire leadership (e.g. 20s) (default 15s)
      --retry-period duration                 Duration the LeaderElector clients should wait between tries of actions (e.g. 5s) (default 2s)
```

Maybe the description doesn't need the example since there is an example in the default value already. The  other places we list examples are when the default value is empty and therefore has no default displayed.

Perhaps the explanation could say the value needs to be a duration string? That's at least what the [go docs](https://golang.org/pkg/time/#ParseDuration) call it.

Reply:
replaced redundant example with "duration string", don't @ me 8)